test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -81,7 +81,7 @@ struct SolidityEndToEndTestExecutionFramework: public SolidityExecutionFramework
 
 BOOST_FIXTURE_TEST_SUITE(SolidityEndToEndTest, SolidityEndToEndTestExecutionFramework)
 
-int constexpr roundTo32(int _num)
+unsigned constexpr roundTo32(unsigned _num)
```

Comment:
Wouldn't using `size_t` make the casts below unnecessary? Or would that just cause warnings in other places?
```suggestion
size_t constexpr roundTo32(size_t _num)
```